lighthouse-core/report/html/renderer/util.js

```diff
@@ -580,6 +580,24 @@ Util.UIStrings = {
 
   /** This label is for a checkbox above a table of items loaded by a web page. The checkbox is used to show or hide third-party (or "3rd-party") resources in the table, where "third-party resources" refers to items loaded by a web page from URLs that aren't controlled by the owner of the web page. */
   thirdPartyResourcesLabel: 'Show 3rd-party resources',
+  /** Label for a row in a data table; entries will be the total number and byte size of all resources loaded by a web page. */
```

Comment:
it's a fuzzy line, but at least for now strings in `util.js` are really only for the report renderer, while these strings are going to go into the raw LHR returned by Lighthouse, so they should really go into `i18n.js`

(I assume moving them so they can be shared between the audits?)